crates/modelardb_compression/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "modelardb_compression"
+version = "0.1.0"
+license = "Apache-2.0"
+edition = "2021"
+authors = ["Soeren Kejser Jensen <devel@kejserjensen.dk>"]
+
+[dependencies]
+arrow = "31.0.0"
+modelardb_common = { path = "../modelardb_common" }
+
+[dev-dependencies]
+proptest = "1.0.0"
+rand = "0.8.5"
```

Comment:
detail: shouldn't there be a newline here?

Reply:
Fixed.